### PR TITLE
Resolve Visibility Issue with Sub-Organization Actions

### PR DESCRIPTION
**Pull request by PasinduYeshan**
### Purpose
Fixed `No option or mechanism provided to switch & Delete to suborganizations that are not present on the first pagination page.` issue.

https://github.com/wso2/identity-apps/assets/61885844/2c8c8fe7-4ba0-4db0-aa40-2ef9efcc6041

### Related Issues
- https://github.com/wso2/product-is/issues/17940

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/organizations/pages/organizations.tsx [hunk 1]
```diff
@@ -101,6 +101,8 @@ const OrganizationsPage: FunctionComponent<OrganizationsPageInterface> = (
     const [ organization, setOrganization ] = useState<OrganizationResponseInterface>(null);
     const [ after, setAfter ] = useState<string>("");
     const [ before, setBefore ] = useState<string>("");
+    const [ authorizedListBefore, setAuthorizedListBefore ] = useState<string>("");
```
Comment (pavinduLakshan) [inline apps/console/src/features/organizations/pages/organizations.tsx:104]: Why do we use "List" in the variable name if they are strings?

File: apps/console/src/features/organizations/pages/organizations.tsx [hunk 2]
```diff
@@ -101,6 +101,8 @@ const OrganizationsPage: FunctionComponent<OrganizationsPageInterface> = (
...
+    const [ authorizedListAfter, setAuthorizedListAfter ] = useState<string>("");
```
Comment (pavinduLakshan) [inline apps/console/src/features/organizations/pages/organizations.tsx:105]: Also, can't we use a single variable and distinguish between before and after cursor using the active page number or any other state variable?

File: apps/console/src/features/organizations/pages/organizations.tsx (hunk 1 above)
Comment (PasinduYeshan) [reply apps/console/src/features/organizations/pages/organizations.tsx:104]: const {
        data: authorizedOrganizationList,
        isLoading: isAuthorizedOrganizationListRequestLoading,
        error: authorizedListFetchRequestError,
        mutate: updateAuthorizedList
    } = useAuthorizedOrganizationsList(
        filterQuery,
        listItemLimit,
        authorizedListAfter,
        authorizedListBefore,
        false
    );
    
Here I used, authorizedListAfter and authorizedListBefore values as before and after for this API call. Therefore, used that variable name.

Comment (PasinduYeshan) [reply apps/console/src/features/organizations/pages/organizations.tsx]: This variables are renamed to `authorizedListPrevCursor` and `authorizedListNextCursor` as discussed.

File: apps/console/src/features/organizations/pages/organizations.tsx (hunk 2 above)
Comment (PasinduYeshan) [reply apps/console/src/features/organizations/pages/organizations.tsx]: Separate variables are required for this to fetch authorized organization list.

**Review by pavinduLakshan — APPROVED** [review]

**Pull request merged**